css/properties/flex.json

```
@@ -46,36 +46,33 @@
                 "version_added": "20",
                 "notes": [
                   "Since Firefox 28, multi-line flexbox is supported.",
-                  "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>."
+                  "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>.",
+                  "Until Firefox 61, flex items that are sized according to their content are sized using <a href='https://drafts.csswg.org/css-sizing-3/#column-sizing'>fit-content, not max-content</a>."
```

While we're tidying up…

```suggestion
                  "Until Firefox 61, flex items that are sized according to their content are sized using <a href='https://drafts.csswg.org/css-sizing-3/#column-sizing'><code>fit-content</code>, not <code>max-content</code></a>."
```